features/admins/admin_post_issues.feature

```diff
@@ -3,9 +3,9 @@ Feature: Admin Actions to Post Known Issues
   As an an admin
   I want to be able to report known issues
 
-  Scenario: Post known issues
-    When I am logged in as an admin
-      And I follow "Admin Posts"
+  Scenario Outline: Post known issues
```

Comment:
Do we want to update these outlines to be like "Authorized admin posts/edits/deletes known issues"?

Reply:
Fewer steps so faster tests, I like it!